### PR TITLE
Replace isAdmin by isSelfHostedAdmin

### PR DESCRIPTION
**Pull request by maxme**

### CODE REVIEW

File: WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsFragment.java [hunk 1]
```diff
@@ -656,7 +656,7 @@ private void initPreferences() {
         }
 
         // hide all options except for Delete site and Enable Location if user is not admin
-        if (!mSite.isAdmin()) hideAdminRequiredPreferences();
+        if (!mSite.isWPCom() && !mSite.isSelfHostedAdmin()) hideAdminRequiredPreferences();
```
Comment (oguzkocer) [inline WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsFragment.java]: This change means that all wpcom users will have access to the admin preferences and I don't think that's what we want. I think it should be something like this:
`if ((!mSite.isWPCom() && !mSite.isSelfHostedAdmin()) || !mSite.getHasCapabilityManageOptions())`

Also, is the "Enable Location if user is not admin" still a valid comment? I didn't quite understand what that means.

Comment (maxme) [inline WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsFragment.java]: Good point, I'll update the code and comment.

Comment (maxme) [inline WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsFragment.java]: Fixed in  	19065a6

**Comment by oguzkocer** [comment]
Looks good :shipit:

**Pull request merged**